src/lint/lint.nim

```diff
@@ -0,0 +1,5 @@
+proc lint*() =
+  stdout.writeLine "No linting rules have yet been implemented.\n" &
+                   "As we're in the process of implementing the linting rules, " &
+                   "please re-run this command regularly to see if your track " &
+                   "passes the latest linting rules."
```

Comment:
```suggestion
proc lint* =
  echo "No linting rules have yet been implemented.\n" &
       "As we're in the process of implementing the linting rules, please " &
       "re-run this command regularly to see if your track passes the " &
       "latest linting rules."
```

Nit: let's prefer `echo` when we want to write to stdout with a newline, and flush.

I use `stdout.write` when I want to write to stdout without a newline (useful when you want to print something to the same line later, possibly based on some condition). And `stdout.writeLine` when writing a lot of lines to stdout, avoiding the overhead of continual, needless flushing (but it can be better to instead fill a string and write that once).